### PR TITLE
Update README.md to address #61

### PR DESCRIPTION
**Pull request by RampedIndent**
Updated README.md to address #61

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,15 @@ fn main() -> Result<(), ::std::io::Error> {
     Ok(())
 }
 ```
+Serde is a required dependancy, and can be added with either the serde_derive crate or serde vrate with feature derive as shown below
+
+```toml
+[dependencies]
+serde = { version = "1.0.152", features = ["derive"] } # <- Only one serde version needed (serde or serd_derive)
```
Comment (matthiasbeyer) [inline README.md:30]:
```suggestion
serde = { version = "1.0.152", features = ["derive"] } # <- Only one serde version needed (serde or serde_derive)
```

File: README.md [hunk 2]
```diff
@@ -23,6 +23,15 @@ fn main() -> Result<(), ::std::io::Error> {
...
+serde_derive = "1.0.152" # <- Only one serde version needed (serde or serd_derive)
```
Comment (matthiasbeyer) [inline README.md:31]:
```suggestion
serde_derive = "1.0.152" # <- Only one serde version needed (serde or serde_derive)
```

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]

**Comment by deg4uss3r** [comment]
Merging/rebasing `main` will fix CI but this does not touch `src/lib.rs` so it is not necessary.

**Comment by RampedIndent** [comment]
New pull request for issue #80

**Pull request closed without merging**